### DISCUSSION TITLE
Missing Credentials For Super User [Aug 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6213] [Discord Threads]
> 🙋‍♂️ **Shashank**

**Comment by o1lab** [comment]
> 🙋‍♂️ **mertm1t**  
#### Hi Shashank, you can use   
#### NC_ADMIN_EMAIL and NC_ADMIN_PASSWORD   
#### environment variables to reset your super admin password (that role is automatically assigned to first user for each instance)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1130153561565704254)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
